eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -101,8 +104,38 @@ void existsInDirWithSpecialSymbolsTest(@TempDir final Path temp) throws IOExcept
         final String decoded = new String(bytes, "CP1252");
         new Home(Paths.get("directory")).save("any content", temp.resolve(decoded));
         MatcherAssert.assertThat(
-            new Home(Paths.get("directory")).exists(temp.resolve(filename)),
-            Matchers.is(true)
+                new Home(Paths.get("directory")).exists(temp.resolve(filename)),
+                Matchers.is(true)
         );
     }
+
+    @Test
+    void loadsBytesFromExistedFile(@TempDir final Path temp) throws IOException {
```

Comment:
typo? Should be "loadBytesFromExistingFile".

Reply:
It was a grammar mistake. Fixed.